src/main/kotlin/me/koddle/tools/SwaggerRouter.kt

```diff
@@ -84,7 +84,16 @@ object SwaggerRouter : KoinComponent {
                         route.handler(
                             OpenAPI3RequestValidationHandlerImpl(op, op.parameters, swaggerFile, swaggerCache)
                         )
-                        route.handler { context -> routeHandler(context, controller, method, op.parameters, roles, opId) }
+                        route.handler { context ->
+                            routeHandler(
+                                context,
+                                controller,
+                                method,
+                                op.parameters,
+                                roles,
+                                opId
+                            )
+                        }
```

Comment:
I really dislike this style, personally. The original easily fits on one line. No need to go from 1 line to 10